docs/source/_ext/probnum-config-options.py

```diff
@@ -0,0 +1,80 @@
+from docutils import nodes
+from docutils.statemachine import StringList
+from sphinx.util.docutils import SphinxDirective, switch_source_input
+
+import probnum as pn
+
+
+class ProbNumConfigOptions(SphinxDirective):
```

Comment:
I dont understand what this class does. It does not seem crucial that I do, but would it make sense either way to document the objective and behaviour, at least with a few one-liner docstrings in the methods?

Reply:
+1 for a short docstring

Reply:
I can add a docstring tomorrow, but I'm not sure how useful this is, since the docstring might be without reading the Sphinx extension tutorial and having read  the tutorial it might be trivial

Reply:
Even just linking to the tutorial could be useful information for future maintainers.

Reply:
Thanks! I agree that even only a link to the tutorial might be useful :)